### PR TITLE
WT-7650 Pull Feed submission Results from Amazon-mws

### PR DESCRIPTION
**Pull request by sachinmaharjan**
Story: https://jira.blurb.com/browse/WT-7650
This will pull the Feed submission Results from Amazon-mws
Need to pass  feed_submission_id

### CODE REVIEW

File: lib/ruby-mws/api/response.rb [hunk 1]
```diff
@@ -2,19 +2,22 @@ module MWS
...
 
         rash = self.new(body)
-        handle_error_response(rash["error_response"]["error"]) unless rash["error_response"].nil?
-        raise BadResponseError, "received non-matching response type #{rash.keys}" if rash["#{name}_response"].nil?
-        rash = rash["#{name}_response"]
-
-        if rash = rash["#{name}_result"]
-          # only runs mods if correct result is present
-          params[:mods].each {|mod| mod.call(rash) } if params[:mods]
+        if !rash.keys.nil? && rash.keys.first == "amazon_envelope"
```
Comment (sachinmaharjan) [inline lib/ruby-mws/api/response.rb]: @pierrel :eyes:

**Comment by pierrel** [comment]
+1

**Comment by tsoo** [comment]
+1

**Comment by sachinmaharjan** [comment]
Merging this

**Pull request merged**